.env.example

```
@@ -4,3 +4,4 @@ NEXT_PUBLIC_API_URL=https://percolator-api-production.up.railway.app
 # Supabase (if needed for bug reporting)
 NEXT_PUBLIC_SUPABASE_URL=
 NEXT_PUBLIC_SUPABASE_ANON_KEY=
+NEXT_PUBLIC_API_URL="https://percolator-api-production.up.railway.app"
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The NEXT_PUBLIC_API_URL key is duplicated

(DuplicatedKey)